src/applications/facility-locator/tests/e2e/errorMessages.cypress.spec.js

```diff
@@ -43,13 +43,27 @@ describe('Facility search error messages', () => {
     cy.get('#facility-type-dropdown').select(
       'Community providers (in VA’s network)',
     );
+    // Wait for services to be saved to state and input field to not be disabled
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(600);
     cy.get('#service-type-ahead-input').focus();
```

Comment:
Try adding `.should('not.be.disabled')` - hopefully we can remove the hard-coded `wait` and the eslint-disable.

Reply:
This worked, thanks!